x/projects/keeper/creation.go

```diff
@@ -82,6 +82,12 @@ func (k Keeper) RegisterKey(ctx sdk.Context, key types.ProjectKey, project *type
 					return utils.LavaError(ctx, k.Logger(ctx), "RegisterKey_add_dev_key_failed", details, "adding developer key failed")
 				}
 			}
+		default:
```

Comment:
yeah, it shouldn't happen since it's checked when the TX is received (just noticed I didn't check this in the addProject TX, so I've added a check). So what do you suggest? omit this case? panic? keep as is?

Reply:
done